pages/experience.tsx

```diff
@@ -100,8 +100,8 @@ userld]); */
 }
 
 Experience.getInitialProps = ({ query }) => {
-  const dataId = query.dataId;
```

Comment:
why this syntax is changed ?

Reply:
it's the same thing. It was probably auto change by es-lint